js/atree_constants.js

```diff
@@ -920,18 +920,12 @@ const atrees = {
             "cost": 1, 
             "display": { "row": 2, "col": 4, "icon": "node_0"},
             "properties": {},
-            "effects": [
-                {
-                    "type": "raw_stat",
-                    "bonuses": [
-                        {
-                            "type": "stat",
-                            "name": "mdPct",
-                            "value": 5
-                        }
-                    ]
-                }
-            ]  
+            "effects": [{
+                "type": "add_spell_prop",
+                "base_spell": 0,
+                "target_part": "Single Shot",
+                "multipliers": [5, 0, 0, 0, 0, 0]
```

Comment:
is our multiplier system going to be
(100 + mult1) / 100 * (100 + mult2) / 100 ... ?

Reply:
no, this is modifying base spell modifiers of the melee attack, if that makes sense

it is additive with other base spell modifiers (double shots / triple shots)